com.github.huluti.ImCompressor.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.github.ImCompressor",
```

Comment:
You've probably forgot to change app id here.

Reply:
Oh... I've forgot to reflect the changes from upstream. Should be better now...